audit-service/maven_output.txt

```diff
@@ -0,0 +1,3188 @@
+Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T20:00:29+01:00)
```

Comment:
This file is immense and also contains personal data like names in the file path. This shouldn't be here.